mino/minogrpc/example_test.go

```diff
@@ -11,6 +14,110 @@ import (
 	"go.dedis.ch/dela/serde"
 )
 
+func TestRpcStreamTree(t *testing.T) {
+	NB_NODES := 200
+
+	nodes := createNodes(NB_NODES, false)
+	rpcs := createRpcs(nodes)
+	exchangeCertificates(nodes)
+	players := createPlayers(nodes)
+	msgs := generateMessages(NB_NODES)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for i, r := range rpcs {
+		go func(i int, rpc mino.RPC) {
```

Comment:
your test will exit without waiting for the messages to be sent/received. You need to wait on your goroutines termination:

```go
wg := sync.WaitGroup{}
wg.Add(NB_NODES)


	for i, r := range rpcs {
		go func(i int, rpc mino.RPC) {
			defer wg.Done()
...

wg.Wait()
```